core/build.gradle

```diff
@@ -91,11 +91,11 @@ dependencies {
     implementation "org.jooq:jooq:${jooqVersion}"
     implementation "software.amazon.awssdk:applicationautoscaling:${awssdkVersion}"
     implementation "software.amazon.awssdk:dynamodb:${awssdkVersion}"
-    implementation "software.amazon.awssdk:core:${awssdkVersion}"
+    implementation "software.amazon.awssdk:core:${awssdkVersion}@pom"
     implementation "org.apache.commons:commons-dbcp2:${commonsDbcp2Version}"
     implementation "mysql:mysql-connector-java:${mysqlDriverVersion}"
     implementation "org.postgresql:postgresql:${postgresqlDriverVersion}"
-    implementation "com.oracle.database.jdbc:ojdbc8-production:${oracleDriverVersion}"
+    implementation "com.oracle.database.jdbc:ojdbc8:${oracleDriverVersion}"
```

Comment:
I tried to specify the `pom` type explicitly for the Oracle JDBC Driver dependency, and it fixed the dependency error, but the integration test failed with the `No suitable driver` error for some reason. So I decided to change the dependency to `com.oracle.database.jdbc:ojdbc8` because we just need the Oracle JDBC Driver.

I referred to the following document: 
https://www.oracle.com/database/technologies/maven-central-guide.html